pkg/apis/core/types.go

```diff
@@ -2970,17 +2976,17 @@ type PodCondition struct {
 	Message string
 }
 
-// PodResizeStatus shows status of desired resize of a pod's containers.
+// Deprecated: PodResizeStatus shows status of desired resize of a pod's containers.
 type PodResizeStatus string
 
 const (
-	// Pod resources resize has been requested and will be evaluated by node.
+	// Deprecated: Pod resources resize has been requested and will be evaluated by node.
 	PodResizeStatusProposed PodResizeStatus = "Proposed"
-	// Pod resources resize has been accepted by node and is being actuated.
+	// Deprecated: Pod resources resize has been accepted by node and is being actuated.
 	PodResizeStatusInProgress PodResizeStatus = "InProgress"
-	// Node cannot resize the pod at this time and will keep retrying.
+	// Deprecated: Node cannot resize the pod at this time and will keep retrying.
 	PodResizeStatusDeferred PodResizeStatus = "Deferred"
-	// Requested pod resize is not feasible and will not be re-evaluated.
+	// Deprecated: Requested pod resize is not feasible and will not be re-evaluated.
```

Comment:
I think you can leave the deprecated tag off of these. I'm not sure these comments are even exported to generated API docs. Just marking the field as deprecated should be enough.

Reply:
took off the deprecated tag. though I think we are still waiting for input from the api reviewer on whether we should just remove these completely

Reply:
@thockin